pegen/c_generator.py

```diff
@@ -492,7 +492,7 @@ def add_var(self, node: NamedItem, names: List[str]) -> Tuple[str, Optional[str]
                     type = rule.type
             elif name.startswith("_loop") or name.startswith("_gather"):
                 type = "asdl_seq *"
-            elif name.startswith("name") or name.startswith("string") or name.startswith("number"):
+            elif name == "name_var" or name == "string_var" or name == "number_var":
```

Comment:
This is better written as `name in ("name_var", ...)`.

Reply:
Right!